script.module.xbmcswift2/lib/xbmcswift2/listitem.py

```diff
@@ -15,16 +15,14 @@ class ListItem(object):
     '''A wrapper for the xbmcgui.ListItem class. The class keeps track
     of any set properties that xbmcgui doesn't expose getters for.
     '''
-    def __init__(self, label=None, label2=None, icon=None, thumbnail=None,
+    def __init__(self, label=None, label2=None, icon=None, thumbnail=None, fanart=None,
```

Comment:
This is a breaking change.

Reply:
Hum, do you recommend to change the addon-id to xbmcswift3 or is it okay to keep it as is? The addons I've been adapting always specify the version in addon.xml for the module

Reply:
I don't care if you break add-ons, just pointing out that this could cause issues. If you move the new argument at the end, it is backwards compatible.

Version is minimum version, not exact or maximum version. Or are you talking about something else?